Src/VimCore/CommandRunner.fs

```diff
@@ -73,6 +73,8 @@ type internal CommandRunner
         | Some count -> count
         | None -> 1
 
+    member x.Inputs = List.rev _data.Inputs

```

Comment:
I always try to remember triple slash comments for members (shows when you hover in the caller).

Reply:
It sees the only place we ever use `CommandRunnerData.Inputs` is this function now. Every use is reversing and allocating a new list. I wonder if we should just change `CommandRunnerData.Inputs` to store the forward list here and avoid the reverse on every access of this member. It would save a few allocations.

Reply:
I wondered about the same.  Not being very familiar with the codebase at this point, I wasn't sure if there was a good reason for stacking the inputs as they came in.

Reply:
I think this code dates back from when I was first writing VsVim and learning F#. I suspect I was attempting to be clever and avoid a full list allocation on every key press by storing the keys in reverse. Then doing a one time reverse once the sequence was completed. Looks like I never actually used that value though 😦

Reply:
I'm just learning F# myself, so I didn't realize lists were singly-linked until digging deeper.  Appending is not only O(n) but also reallocates the original list.  That's not what I expected!
I think we're talking about very small lists in any realistic scenario (that I can think of), but keeping the inputs in order of entry also makes more semantic sense to me.

As soon as I get a chance, I'll take care of this detail and submit another PR.